src/applications/disability-benefits/686c-674/components/IntroductionPageFormProcess.jsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+
+export const IntroductionPageFormProcess = () => (
+  <>
+    <h4>Follow the steps below to apply to add or remove a dependent.</h4>
```

Comment:
Decided to break this out to make the introduction page a little more manageable.